kokoro/scripts/build/build_package.sh

```diff
@@ -42,24 +42,54 @@ export_to_sponge_config "PACKAGE_VERSION" "${PKG_VERSION}"
 
 ARTIFACT_REGISTRY="us-docker.pkg.dev"
 sudo docker-credential-gcr configure-docker --registries="${ARTIFACT_REGISTRY}"
-CACHE_LOCATION="${ARTIFACT_REGISTRY}/stackdriver-test-143416/google-cloud-ops-agent-build-cache/ops-agent-cache:${DISTRO}"
+function get_cache_location() {
+  echo "${ARTIFACT_REGISTRY}/stackdriver-test-143416/google-cloud-ops-agent-build-cache/ops-agent-cache$1:${DISTRO}"
```

Comment:
can we make this defaulting happen elsewhere? or if it's not actually invoked, remove it?